tests/Application/package.json

```diff
@@ -1,52 +1,63 @@
 {
```

Comment:
A lot of deps here :D

Should probably look a bit more like https://github.com/Setono/SyliusPluginSkeleton/blob/master/tests/Application/package.json